server/routes.js

```diff
@@ -83,12 +83,12 @@ module.exports = (app) => {
   app.get('/github/apply', mw.ga, mw.addTitle('Sign up your Github repository'), render);
   app.get('/connect/github', mw.ga, render);
   app.get('/:slug/connect/twitter', mw.ga, render);
-  app.get('/:slug/edit-twitter', mw.ga, mw.fetchGroupBySlug, render);
+  app.get('/:slug/edit-twitter', mw.ga, controllers.profile, render);
```

Comment:
Ideally, the controller should call render.
(but it was already like that, it's ok to keep it that way for now if you want)